app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/learning/LearningFragment.kt

```diff
@@ -216,11 +245,13 @@ class LearningFragment : Fragment() {
      */
     private fun dropCallback(event: DragEvent, v: View): Boolean {
```

Comment:
looks llike event is not used in this method

Reply:
yup, it's a relique from a long forgotten past